tools/datagen/src/bin/datagen.rs

```diff
@@ -370,6 +373,14 @@ fn export_cldr<'data>(
 
     let keys = get_all_cldr_keys();
 
+    let keys = if let Some(allowed_keys) = allowed_keys {
+        keys.into_iter()
+            .filter(|k| allowed_keys.contains(&*k.to_string()))
```

Comment:
Nit (optional): `k.writeable_to_string()` instead of `k.to_string()`.

We could also probably avoid allocating a `String` entirely, but that complexity is not worth it in a build-time tool.

Reply:
Yeah it's hard to do that without adding more stuff to ResourceKey